packages/evm-adapter/src/constants.ts

```diff
@@ -3,13 +3,17 @@ import { BigNumber, utils } from 'ethers';
 export enum ChainId {
   ETHEREUM_MAINNET = 1,
   GOERLI = 5,
+  SEPOLIA = 11155111,
   POLYGON_MAINNET = 137,
   POLYGON_MUMBAI = 80001,
 }
 
 export enum ContractAddress {
-  POLYGON_MUMBAI = '0x1B43FB8a3c4CC150768C72656C352bdf35Dd1209',
   POLYGON_MAINNET = '0x644cF27D0CB17cf252BCfb96e2821fc17109761f',
+  POLYGON_MUMBAI = '0x1B43FB8a3c4CC150768C72656C352bdf35Dd1209',
+  ETHEREUM_MAINNET = '0x1B43FB8a3c4CC150768C72656C352bdf35Dd1209',
+  GOERLI = '0x1B43FB8a3c4CC150768C72656C352bdf35Dd1209',
```

Comment:
maybe best to prefix it with ETHEREUM so we know it's eth related

Reply:
OK, just like with poly, make sense